storage/mysql/storage_test.go

```diff
@@ -292,6 +263,14 @@ func createTree(db *sql.DB, tree *trillian.Tree) (*trillian.Tree, error) {
 	return tree, nil
 }
 
+func createTreeOrPanic(db *sql.DB, create *trillian.Tree) *trillian.Tree {
+	tree, err := createTree(db, create)
+	if err != nil {
+		panic(fmt.Sprintf("Error creating tree: %v", err))
```

Comment:
Might make more sense to pass in `*testing.T` and calling `t.Fatalf` here. Same for other panicking function in this file.

Reply:
Not in the scope of this PR though.

Reply:
Removing panics would be better. For the moment this should probably be called mustCreateTree() or similar.

Reply:
Not sure I follow. By removing panics do you mean the following?
```
if err != nil {
	fmt.Sprintf("Error creating tree: %v", err)
}
```
... or removing panics and adding `t.Fatalf` as I suggested?